src/main/java/org/breedinginsight/daos/ProgramUserDAO.java

```diff
@@ -115,6 +115,7 @@ public ProgramUser getProgramUser(UUID programId, UUID userId) {
         Result<Record> records = getProgramUsersQuery(createdByUser, updatedByUser)
                 .where(PROGRAM_USER_ROLE.USER_ID.eq(userId))
                 .and(PROGRAM.ACTIVE.eq(true))
+                .and(PROGRAM_USER_ROLE.ACTIVE.eq(true))
```

Comment:
I'd suggest renaming this method to `getActiveProgramUser` (and making an additional method to fetch all users) so that it is clear that a user could exist, but be inactive, and therefore nothing would be returned by this method.

Reply:
I removed that new line, so it returns all program users now. This was a quick addition I thought was the fix, but it was not.